spec/models/concerns/solidus_stripe/money_to_stripe_amount_converter_spec.rb

```diff
@@ -7,44 +7,44 @@
   describe '.to_stripe_amount' do
     it 'converts to the fractional expected by stripe' do
       (described_class::ZERO_DECIMAL_CURRENCIES - %w[MGA]).each do |currency|
-        expect([currency, described_class.to_stripe_amount(12345, currency).to_i]).to eq([currency, 12345])
```

Comment:
I'm fine with the change, but just to give some context I think it was like that so that errors would include a reference to the currency when inside loops. If you think it's a worthy feature maybe it's better to add a comment for the next passerby wondering why it's done in this weird way 🤪

Reply:
Oh, thanks for clarifying. I've added another commit using a custom message for the failure and adding the currency.